### PR TITLE
Proofreading dev documents under /user_doc/guide/installation

### PR DESCRIPTION
**Pull request by Tianqi-Dotes**
Proofreading dev documents under /user_doc/guide/installation

### CODE REVIEW

File: docs/en-us/dev/user_doc/guide/installation/docker.md [hunk 1]
```diff
@@ -328,29 +326,30 @@ $ docker build --build-arg VERSION=1.3.8 -t apache/dolphinscheduler:1.3.8 .
...
 1. Support [docker buildx](https://docs.docker.com/engine/reference/commandline/buildx/)
-2. Own the push permission of https://hub.docker.com/r/apache/dolphinscheduler (**Be cautious**: The build command will automatically push the multi-platform architecture images to the docker hub of apache/dolphinscheduler by default)
+2. Own the push permission of `https://hub.docker.com/r/apache/dolphinscheduler` (**Be cautious**: The build command will automatically push the multi-platform architecture images to the docker hub of `apache/dolphinscheduler` by default)
 
 Execute:
 
 ```bash
 $ docker login # login to push apache/dolphinscheduler
-$ bash ./docker/build/hooks/build
+$ bash ./docker/build/hooks/build x
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/installation/docker.md:338]: Is it add `x` by mistake?
```suggestion
$ bash ./docker/build/hooks/build
```

Comment (Tianqi-Dotes) [reply docs/en-us/dev/user_doc/guide/installation/docker.md:338]: The 'x' is right. The tool is called 'buildX' the command is 'build x' and I have checked the origin document. It's fine.

Comment (zhongjiajie) [reply docs/en-us/dev/user_doc/guide/installation/docker.md:338]: oh, my bad, thanks for your clarification

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**